macros/schema_tests/expression_is_true.sql

```diff
@@ -12,7 +13,7 @@ validation_errors as (
     select
         *
     from meet_condition
-    where not({{expression}})
+    where not( {{ expression if column_name is none else [column_name, expression] | join(' ')  }})
```

Comment:
Hmm this is a little hard to read for me, I think I'd prefer an `if` statement:
```suggestion
{% if column_name is none %}
where not({{ expression }})
{% else %}
where not({{ column_name }} {{ expression }})
{% endif %}
```